### ISSUE TITLE
Implement npm run test

### ISSUE THREAD

**Issue by ajvincent** [agrublev/fc-filepick#15]
@agrublev I've started a fork of this project, with the intent of generating some pull requests to improve this project somewhat.  First on my docket is adding some tests, so future pull requests don't break anything...

I'm wondering if you have a preference for a testing framework (Jasmine, Jest, Mocha).  I'd want to write tests and code in accordance with your preferences.

**Comment by agrublev** [comment]
My personal favorite is Jest

**Comment by agrublev** [comment]
@ajvincent  I take it you lost interest?

**Comment by ajvincent** [comment]
Yes and no.  I wouldn't mind doing this, but when I initially filed this issue, I was unemployed.  Since then I have been hired, and I don't have the spare time I used to.

Really, at the time, I was waiting for a decision from you.  It seemed wasteful to implement in one framework when there was a very good chance you'd pick another.
